src/atoms/social/speaker/presentational.js

```diff
@@ -13,29 +13,39 @@ const listStyles = {
   padding: 0,
   fontSize: 20
 };
+
 const listItemStyles = {
-  padding: '0 0 0 .5em',
+  padding: '0 0 0 1em',
   height: iconSize,
   span: {
     paddingLeft: '.25em'
   }
 };
 
+const socialItemStyle = {
+  display: 'flex',
+  alignItems: 'center'
+};
+
 export default function SpeakerSocialIcons({twitter, github}) {
   const nonHoveredIconColor = '#888';
   const TwitterIcon = hovered => (
     <li css={listItemStyles}>
       <ExternalLink to={`https://twitter.com/${twitter}`}>
-        <Icon fill={hovered ? '#1DA1F2' : nonHoveredIconColor} size={iconSize} name="twitter" />
-        <span>{twitter}</span>
+        <div css={socialItemStyle}>
```

Comment:
i've already been considering having `ExternalLink` take a `css` prop that is passed to the underlying `<a>` DOM element for some other stuff. i think i'd prefer going that route over adding more DOM elements just for styling

Reply:
i've been thinking about this some more. i still think it would be best to apply the styling to the external link, but i'm second guessing myself on the "how".

i tend be on the fence early on with most projects that encapsulate styles within components between the options of either allowing styles to be passed from a parent vs making the components more rigid and managing their own "variants". passing styles from the parent is easier early on, but i always regret breaking that encapsulation later. the variant approach takes a little more discipline, but that tends to work out better in the long run anyway by fighting the temptation to have a whole bunch of variations of the same thing throughout the site/app.

i'm even more on the fence in this case because i kind of see the current design as temporary since we've considered a redesign after getting transitioned over to gatsby. if we do follow through on that i'd expect we'd put more effort into a [design system](https://github.com/dsmjs/components/issues/88) and would be wanting a more disciplined approach. that thought makes taking the easier route attractive at this point, but seeing how long the transition has taken, i don't have high confidence that a redesign would follow quickly afterward.

thoughts?

Reply:
Ended up taking internal links in this direction in order to handle existing variations: https://github.com/dsmjs/components/commit/732572fdd17704a6a89e8db4d1760406b889bc7a

I think this makes me lean further in this direction for external links